langchain/langchain_interceptor.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import annotations
+import asyncio
+from contextvars import copy_context
+
+from langsmith import get_tracing_context
```

Comment:
Importing non-stdlib things as non-passed-through will increase per-workflow memory. Can we move this to inside pass through block?